.rubocop.yml

```diff
@@ -234,32 +252,70 @@ Minitest/AssertInDelta:
   Enabled: true
 Minitest/AssertKindOf:
   Enabled: true
+Minitest/AssertOperator:
+  Enabled: false
 Minitest/AssertOutput:
   Enabled: true
 Minitest/AssertPathExists:
   Enabled: true
+Minitest/AssertPredicate:
+  Enabled: false
+Minitest/AssertRaisesWithRegexpArgument:
+  Enabled: true
+Minitest/AssertSame:
+  Enabled: true
 Minitest/AssertSilent:
   Enabled: true
 Minitest/AssertWithExpectedArgument:
   Enabled: true
 Minitest/AssertionInLifecycleHook:
   Enabled: true
+Minitest/DuplicateTestRun:
+  Enabled: true
+Minitest/EmptyLineBeforeAssertionMethods:
+  Enabled: false
+Minitest/LifecycleHooksOrder:
+  Enabled: true
 Minitest/LiteralAsActualArgument:
   Enabled: true
 Minitest/MultipleAssertions:
   Enabled: true
+Minitest/NonExecutableTestMethod:
+  Enabled: true
+Minitest/NonPublicTestMethod:
+  Enabled: true
+Minitest/RedundantMessageArgument:
+  Enabled: true
 Minitest/RefuteInDelta:
   Enabled: true
 Minitest/RefuteKindOf:
   Enabled: true
+Minitest/RefuteOperator:
+  Enabled: false
```

Comment:
Current:
```ruby
assert before_time <= @site.time
```
After style change (with `AssertOperator` counterpart):
```ruby
assert_operator before_time, :<=, @site.time
```

The new style is objectively harder for humans to parse. So, `disabled` without veto. (applied to counterpart `RefuteOperator` as well).

Reply:
I see. I think the idea is to have better error messages since `assert boolean_expr` makes test output pretty useless. Perhaps we should enforce that `assert` calls must have a message component, i.e. `assert boolean_expr, message_explaining_expectation`. If it's not already a cop, then no worries.